dashboard/src/components/TabsBar.tsx

```diff
@@ -0,0 +1,50 @@
+/**
+ * @file TabsBar.tsx
+ *
+ * This component is the bar that contains the tabs.
+ * it gets the tabs as a prop that contains a list with the name of the tabs
+ * and the component that should be rendered when the tab is clicked.
+ *
+ * @param {Array} tabs - the tabs that should be rendered
+ * @param {string} activeTab - the name of the active tab
+ * @param {Function} setActiveTab - the function that should be called when a tab is clicked
+ * @param {Function} setTabContent - the function that should be called when a tab is clicked
+ *
+ * @returns {JSX.Element} - the tabs bar
+ *
+ *
+ */
+
+import React from "react";
+import "../index.css";
```

Comment:
i think u only need to load this once in App.js

Reply:
It is possible, I'll try to delete it and see what happens.